registration.html

```diff
@@ -245,7 +245,7 @@ <h3 class="slim-title fineprint" for="en-lahpriv">By submitting your application
 		</section>
 	</main>
 
-	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+	<script src="/js/jquery.min.js"></script>
```

Comment:
adding a `/` before `js` is pointless and creates issues for people developing locally. Please remove this from index and registration